GenderPayGap.WebUI/Models/Report/NewReportOverviewViewModel.cs

```diff
@@ -0,0 +1,189 @@
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Female employees in lower quarter",
+            NameWithinSentence = "female employees in lower quarter")]
+        [GovUkValidateDecimalRange(Minimum = 0, Maximum = 100)]
+        public decimal? FemaleLowerPayBand { get; set; }
+
+        #endregion
+
+        #region Hourly
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Difference in hourly pay (mean)",
+            NameWithinSentence = "difference in hourly pay (mean)")]
+        public decimal? DiffMeanHourlyPayPercent { get; set; }
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Difference in hourly pay (median)",
+            NameWithinSentence = "difference in hourly pay (median)")]
+        public decimal? DiffMedianHourlyPercent { get; set; }
+
+        #endregion
+
+        #region Bonus Percent
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Females who received bonus pay",
+            NameWithinSentence = "females who received bonus pay")]
+        [GovUkValidateDecimalRange(Minimum = 0, Maximum = 100)]
+        public decimal? FemaleBonusPayPercent { get; set; }
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Males who received bonus pay",
+            NameWithinSentence = "males who received bonus pay")]
+        [GovUkValidateDecimalRange(Minimum = 0, Maximum = 100)]
+        public decimal? MaleBonusPayPercent { get; set; }
+
+        #endregion
+
+        #region Bonus Mean/Median
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Difference in bonus pay (mean)",
+            NameWithinSentence = "difference in bonus pay (mean)")]
+        public decimal? DiffMeanBonusPercent { get; set; }
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Difference in bonus pay (median)",
+            NameWithinSentence = "difference in bonus pay (median)")]
+        public decimal? DiffMedianBonusPercent { get; set; }
+
+        #endregion
+
+        #region Person responsible
+
+        public string ResponsiblePersonFirstName { get; set; }
+        public string ResponsiblePersonLastName { get; set; }
+        public string ResponsiblePersonJobTitle { get; set; }
+
+        #endregion
+
+        #region Org size
+
+        public OrganisationSizes? SizeOfOrganisation { get; set; }
+
+        #endregion
+        
+        #region Link to GPG Info
+
+        public string LinkToOrganisationWebsite { get; set; }
+
+        #endregion
+        
+        public bool OptedOutOfReportingPayQuarters { get; set; }
+        
+        public string GetPayQuarterValue(decimal? payQuarterValue)
+        {
+            return OptedOutOfReportingPayQuarters ? "Not Applicable" : GetPercentageValue(payQuarterValue);
+        }
+
+        public string GetPercentageValue(decimal? value)
+        {
+            return value == null ? "Not Completed" : $"{value}%";
```

Comment:
there are some cases where "Not applicable" should be displayed:
* if opted out of reporting pay quarters
* if % of men who received bonus pay is 0 -> mean and median bonus pay can't be calculated

Reply:
What if the % of women is 0?